.github/workflows/tests.yml

```diff
@@ -128,30 +128,26 @@ jobs:
       - name: List 3rd-party stub dependencies installed
         run: uv pip freeze
       - name: Get pyright version
-        uses: SebRollen/toml-action@v1.0.2
-        id: pyright_version
-        with:
-          file: "pyproject.toml"
-          field: "tool.typeshed.pyright_version"
+        run: echo PYRIGHT_VERSION=$(pyright --version | cat | head -n 1 | cut -d " " -f 2) >> $GITHUB_ENV
```

Comment:
The CI issue is simple, the version obtention isn't working so it's fetching the latest version

![image](https://github.com/python/typeshed/assets/1350584/71091dde-576f-42f0-9bd2-4c13ae3843dd)

Sidenote, I think I preferred using the step output `$GITHUB_OUTPUT`, which also keeps the rest of the file the same?

Reply:
Yeah, this evaluates to the empty string, which I'm pretty sure I can't distinguish in the action (as empty string is the default value I have to apply to have a parameter of type string for the action).

Reply:
Maybe to prevent this issue, I should just default it to the string "latest" instead, and hope that nobody breaks downstream...

Reply:
Even with the string "latest", the CI would still fail. Typeshed isn't ready for pyright>=1.1.351 . That's what the CI failures are.

Reply:
The issue is specifically that `{{ env.PYRIGHT_VERSION }}` is empty, so that's taken as the default, which is latest, but I think anyone that explicitly passes an empty string into the action probably didn't mean to do that. So, I was just thinking aloud whether or not I could detect this and warn. No action required here!